internal/grid/connection.go

```diff
@@ -615,10 +616,6 @@ func (c *Connection) sendMsg(conn net.Conn, msg message, payload msgp.MarshalSiz
 	if c.outgoingBytes != nil {
 		c.outgoingBytes(int64(len(dst)))
 	}
-	err = conn.SetWriteDeadline(time.Now().Add(connWriteTimeout))
-	if err != nil {
-		return err
-	}
```

Comment:
its not needed since the caller already provides the context @klauspost it can be retained I don't see any harm as such but it was just something that i thought is not needed.

let me add it back.